bitnami/metallb/Chart.yaml

```diff
@@ -30,4 +30,4 @@ sources:
   - https://github.com/metallb/metallb
   - https://github.com/bitnami/bitnami-docker-metallb
   - https://metallb.universe.tf
-version: 2.2.0
+version: 2.2.1
```

Comment:
Since you are adding a new parameter you should bump the minor version

Reply:
Done